Dashboard/Views/MainWindowView.axaml

```diff
@@ -0,0 +1,60 @@
+﻿<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:views="clr-namespace:Dashboard.Views"
+        xmlns:vm="clr-namespace:Dashboard.ViewModels"
+        xmlns:models="clr-namespace:Dashboard.Models"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="Dashboard.Views.MainWindowView"
+        x:DataType="vm:MainWindowViewModel"
+        Title="MainWindow"
+        Foreground="Black"
+        Background="#c4c4c4">
+        <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:MainWindowViewModel />
+    </Design.DataContext>
+    
+    <Grid RowDefinitions="Auto, *, Auto" >
+        
+        <Grid Grid.Row="0" Height="100" >
+                <Image Source="/Assets/nuvotion_logo.png" Width="100" Height="100" HorizontalAlignment="Right"
+                       VerticalAlignment="Top" RenderTransformOrigin="0.5,0.5">
+                        <Image.RenderTransform>
+                            <RotateTransform Angle="90" />
+                        </Image.RenderTransform>
+                    </Image>
+                    <TextBlock> Nuvotion Motorsports</TextBlock>
+                    <!-- Menu to navigate between pages -->
+                    <ListBox Name="NaviBar" VerticalAlignment="Bottom" ItemsSource="{Binding Items}"
+                             SelectedItem="{Binding SelectedListItem}">
+                        <ListBox.Styles>
+                            <Style Selector="ListBoxItem">
+                            </Style>
+                        </ListBox.Styles>
+                        <ListBox.ItemsPanel>
+                            <ItemsPanelTemplate>
+                                <StackPanel Orientation="Horizontal">
+                                </StackPanel>
+                            </ItemsPanelTemplate>
+                        </ListBox.ItemsPanel>
+                        <ListBox.ItemTemplate>
+                            <DataTemplate DataType="{x:Type models:ListItemTemplate}">
+                                <TextBlock Text="{Binding Label}"></TextBlock>
+                            </DataTemplate>
+                        </ListBox.ItemTemplate>
+                    </ListBox>
+        </Grid>
+            
+    <Grid Grid.Row="1">
+        <Viewbox Stretch="Uniform" StretchDirection="Both">
+        <ContentControl Name="MainContent" Content="{Binding CurrentPage}"/> 
```

Comment:
I know originally you were thinking of going with the ViewLocator, what you've done now is awesome and works really well but whats the reasoning for the change?

Reply:
So the main reason I went with this method was because I was having issues with ViewLocator getting DataContext to each View so they wouldn't get the logic they needed (DataView wouldn't display the data etc). After some digging I found that some of the Avalonia contributors recommended this so I went with it.